models/index.js

```diff
@@ -0,0 +1,30 @@
+const Sequelize = require('sequelize')
+const allConfigs = require('../configs/sequelize')
+const monthsModel = require('./months')
+const moonsModel = require('./moons')
+const activitiesModel = require('./activities')
+
+const environment = process.env.NODE_ENV ? process.env.NODE_ENV : 'development'
+const config = allConfigs[environment]
+
+const connection = new Sequelize('witches', 'witches', 'w!tChp1e@$3', {
+  host: 'localhost', dialect: 'mysql'
+})
+
+const Months = monthsModel(connection, Sequelize)
+
+const Moons = moonsModel(connection, Sequelize, Months)
+
+const Activities = activitiesModel(connection, Sequelize, Months, Moons)
+
+Months.hasMany(Moons)
+Months.hasMany(Activities)
+Moons.belongsTo(Months)
+Activities.belongsTo(Months)
```

Comment:
You set up your two way relationship between months and moons here but activities is related to both and you only set up the Months side of that.

```JavaScript
Months.hasMany(Moons)
Moons.belongsTo(Months)

Months.hasMany(Activities)
Activities.belongsTo(Months)

Moons.hasMany(Activities)
Activities.belongsTo(Moons)
```